axoproject/src/tests.rs

```diff
@@ -15,7 +24,7 @@ fn test_self_detect() {
     assert_eq!(project.kind, WorkspaceKind::Rust);
     assert_eq!(packages.len(), 3);
 
-    let package = packages[0].1;
+    let package = packages[1].1;
```

Comment:
is there a way to more expressively write this (or comment why it's choosing things this way?)

Reply:
When I spoke to Gankra, she mentioned her original goal with these tests was to test that package order was somewhat stable - hence accessing these with indices instead of by name. Since we've updated how we're fetching packages, it has shuffled the package order returned by axoproject so the original assumptions weren't relevant anymore.